job-dsls/jobs/deploy_jobs.groovy

```diff
@@ -90,11 +90,7 @@ def final REPO_CONFIGS = [
         "kie-uberfire-extensions"   : [
                 timeoutMins            : 40,
                 ircNotificationChannels: ["#guvnordev"],
-                downstreamRepos        : ["guvnor"]
-        ],
-        "guvnor"                    : [
-                ircNotificationChannels: ["#guvnordev"],
-                downstreamRepos        : ["kie-wb-playground"]
+                downstreamRepos        : []
```

Comment:
Sorry, should have been more clear. This needs to be `downstreamRespo: [kie-wb-playground]`.